.eslintrc.js

```diff
@@ -1 +1,11 @@
-module.exports = require('./index.js');
+const t = [require('./index.js')];
+
+module.exports = {
+  extends: require.resolve('./index.js'),
+  rules: {
+    // Forced to disable rule for eslint-config-eko as it behave weird.
+    // Somehow it reports all exports across package.
+    // Might be removed if we find out that it behave like this everywhere.
+    'import/no-unused-modules': 'off',
```

Comment:
This rule should actually not report when the file has at least one export?

Reply:
@tavurth 
```
- modules without any exports
- individual exports not being statically imported or requireed from other modules in the same project
```

Reply:
@tavurth 
I reverted `.eslintrc.js` as this code not required (I was forced to disable `import/no-unused-modules`)
https://github.com/EkoCommunications/EkoESLintConfig/pull/6/files#diff-17ec2f3ce30d31d97dd608a120ada07fR14